src/rollup/RollupCore.sol

```diff
@@ -77,6 +77,7 @@ abstract contract RollupCore is IRollupCore, PausableUpgradeable {
     uint64 internal constant GENESIS_NODE = 0;
 
     bool public validatorWhitelistDisabled;
+    address public anyTrustFastConfirmer;
```

Comment:
should we move this below `_hostChainIsArbitrum` to keep storage layout the same